Software/Latex Docs/Software Steps/Software Steps.tex

```diff
@@ -92,13 +92,11 @@ \section{Flashing the Arduino Code}
 \item Clone the code repo:
 	\begin{enumerate}
 	\item git clone https://github.com/nasa-jpl/osr-rover-code.git
-	\item git checkout osr-ROS
-	\item git pull
 	\end{enumerate}
 
 \item Build our custom library:
 	\begin{enumerate}
-	\item Select the downloaded Arduino folder and create a .ZIP file from it
+	\item Select the downloaded Arduino folder from within the osr-rover-code repo and create a .ZIP file from it
```

Comment:
Remember that our target audience here is people who may not have any terminal experience. Please add the relevant terminal commands for how to zip these files and rename it as necessary.